CONTRIBUTING.md

```diff
@@ -18,7 +18,7 @@ cd ocaml-api-watch
 And setup a local opam switch to install the right ocaml version along with the
 set of base dependencies:
 ```
-opam switch create ./ 4.12.1 --deps-only -t
+opam switch create ./ 4.14.0 --deps-only -t
```

Comment:
I suggest we just remove specification of the compiler version here, since it is already specified in the `dune-project` file at https://github.com/nathanreb/ocaml-api-watch/blob/59eeb9e4c7e09f344cba7df64313aa1b3c8b5768/dune-project#L18 and by no repeating the version here, we avoid having to keep this (untested) instruction up to date :)

```suggestion
opam switch create ./ --deps-only -t
```

Reply:
Oh that sounds even better! I've updated to match.